samples/charts/category-chart/marker-options/Pages/CategoryChartMarkerOptions.razor

```diff
@@ -0,0 +1,191 @@
+﻿@page "/CategoryChartMarkerOptions"
+
+@using Infragistics.Blazor.Controls
+@inject IInfragisticsBlazor InfragisticsBlazor;
+@inject IJSRuntime JSRuntime;
+
+<div class="igContainer">
+    <div class="igOptions">
+        <span className="igOptions-label">Chart Type: </span>
+        <select @onchange="OnChartTypeChanged">
+            <option>Auto</option>
+            <option>Area</option>
+            <option>Column</option>
+            <option>Point</option>
+            <option>Line</option>
+            <option>Spline</option>
+            <option>SplineArea</option>
+            <option>StepArea</option>
+            <option>StepLine</option>
+            <option>Waterfall</option>
+        </select>
+        <span className="igOptions-label"> Marker Type: </span>
+        <select @onchange="OnMarkerTypeChanged">
+            <option>Automatic</option>
+            <option>Circle</option>
+            <option>Triangle</option>
+            <option>Pyramid</option>
+            <option>Square</option>
+            <option>Diamond</option>
+            <option>Pentagon</option>
+            <option>Hexagon</option>
+            <option>Tetragram</option>
+            <option>Pentagram</option>
+            <option>Hexagram</option>
+            <option>None</option>
```

Comment:
perpend all values of options with "MarkerType." this way, it will simply OnMarkerTypeChanged()